src/test/java/com/wikia/webdriver/testcases/mobilewikitests/TrackingOptInModalTests.java

```diff
@@ -26,7 +26,7 @@
   @Execute(asUser = User.ANONYMOUS, trackingOptIn = false)
   @Test(groups = {"mobile-wiki-tracking-opt-in"},
       dataProviderClass = TrackingOptInDataProvider.class,
-      dataProvider = "GDPRcountries"
+      dataProvider = "GDPRCountriesForTest"
```

Comment:
@qaga can you setup separate tests for full list so we can run them periodically?

Reply:
yes I will setup them but in separate PR if that's OK so that change can go to prod ASAP